pydantic_extra_types/color.py

```diff
@@ -14,6 +14,12 @@
 from colorsys import hls_to_rgb, rgb_to_hls
 from typing import Any, Callable, Tuple, Union, cast
 
+try:
+    from typing import Literal
+except ImportError:  # pragma: no cover
+    # Python <3.8
+    from typing_extensions import Literal  # type: ignore[assignment]
```

Comment:
Can you check the Python version instead?

Reply:
Done - had to add another `pragma: no cover` for 3.7 to be happy, but I can see the benefit.